src/__tests__/LDClient-test.js

```diff
@@ -407,6 +407,23 @@ describe('LDClient', () => {
   });
 
   describe('identify', () => {
+    it('does not set user until the flag config has been updated', async () => {
```

Comment:
Probably need a better test. What we really want to assert is that the user and the flags are updated synchronously at the same time.

Reply:
I don't think "synchronously" is a meaningful term in this context— I get what you mean, but there really isn't any way to verify that they're updated on the same pass through the event loop rather than one tick apart or whatever. I'm not super happy with the arbitrary 200ms sleep but right now I can't think of a better way.